Cooper/ClientApp/src.v2/app/services/language-storage/language-storage.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LanguageStorageService {
+  getLanguage(): string {
+    if (localStorage) {
+        return localStorage.getItem('language') || 'en';
+    } else {
+        return 'en';
+    }
+}
+
+setLanguage(language: string) {
```

Comment:
SetCurrentLanguage or setApplicationLanguage

Reply:
A little mistake in word:  `defoult` -> `default`